plans/network/main.go

```diff
@@ -210,41 +176,23 @@ func pingpong(runenv *runtime.RunEnv) error {
 		}
 		runenv.RecordMessage("ping RTT was %s [%s, %s]", rtt, rttMin, rttMax)
 
+		// Don't reconfigure the network until we're done with the first test.
 		state := sync.State("ping-pong-" + test)
+		client.MustSignalAndWait(ctx, state, runenv.TestInstanceCount)
 
-		// Don't reconfigure the network until we're done with the first test.
-		_, err = client.SignalEntry(ctx, state)
-		if err != nil {
-			return err
-		}
-		err = <-client.MustBarrier(ctx, state, runenv.TestInstanceCount).C
-		if err != nil {
-			return err
-		}
 		return nil
 	}
-	err = pingPong("200", 200*time.Millisecond, 210*time.Millisecond)
+	err = pingPong("200", 200*time.Millisecond, 215*time.Millisecond)
 	if err != nil {
 		return err
 	}
 
 	config.Default.Latency = 10 * time.Millisecond
```

Comment:
Offtopic, but this is a bit too granular to measure consistently in order environments, but locally.

Reply:
Yup, we should probably set latencies that are orders of magnitude different, so that our leniency ranges can also be laxer.